python/ray/serve/http_proxy.py

```diff
@@ -32,8 +32,9 @@ async def fetch_config_from_controller(self, name, controller_name):
         self.route_table = await controller.get_router_config.remote()
 
         self.request_counter = metrics.Count(
-            "num_http_requests", "The number of HTTP requests processed",
-            "requests", ["route"])
+            "num_http_requests",
+            description="The number of HTTP requests processed",
+            tag_keys=("route"))
```

Comment:
does this need to be a tuple?

Reply:
I currently have logic to convert a string to tuple (when key is provided this way), but I think you are right. We should always only allow tuple. I will change it to ("route", ) and have some runtime type checking.